lib/web/apiserver.go

```diff
@@ -4092,7 +4093,7 @@ func (h *Handler) AuthenticateRequest(w http.ResponseWriter, r *http.Request, ch
 // ProxyWithRoles returns a reverse tunnel proxy verifying the permissions
 // of the given user.
 func (h *Handler) ProxyWithRoles(ctx *SessionContext) (reversetunnelclient.Tunnel, error) {
-	accessChecker, err := ctx.GetUserAccessChecker()
+	accessChecker, err := ctx.GetAccessChecker()
```

Comment:
It feels pretty unclear where one should user GetAccessChecker or GetUserAccessChecker, and even though I know exactly what each of them is doing, I still have no idea why e.g. this function prefers GetAccessChecker